pydy/tests/test_system.py

```diff
@@ -395,3 +397,67 @@ def test_integrate(self):
         sys = System(self.kane, times=times)
         with testing.assert_raises(NotImplementedError):
             sys.generate_ode_function(generator='made-up')
+
+    def test_joint_system_init(self):
+        sys = System(method='joints')
```

Comment:
I think its a great idea and we should definitely think about it. I will finish up the unit tests, until then all of us can think about the pro and cons of this implementation.